static/src/javascripts/projects/commercial/modules/messenger/passback.ts

```diff
@@ -213,6 +213,13 @@ const init = (register: RegisterListener): void => {
 											`Passback: setting height of passback slot to ${slotHeight}`,
 										);
 										slotElement.style.height = slotHeight;
+
+										// Also resize the initial outstream iframe so
+										// it doesn't block text selection directly under
+										// the new ad
+										iframe.style.height = slotHeight;
+										iFrameContainer.style.height =
+											slotHeight;
```

Comment:
Another solution might be to remove `.ad-slot__content` completely but I don't understand the passback mechanism well enough to know if this will have unintended consequences.

Something like:

```suggestion
										iFrameContainer.remove();
```